util/testutil/workers/util_windows.go

```diff
@@ -0,0 +1,79 @@
+package workers
+
+import (
+	"bytes"
+	"context"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"time"
+
+	"github.com/moby/buildkit/util/testutil/integration"
+)
+
+func requireRoot() error {
+	return nil
+}
+
+func runBuildkitd(
+	ctx context.Context,
+	conf *integration.BackendConfig,
+	args []string,
+	logs map[string]*bytes.Buffer,
+	uid, gid int,
+	extraEnv []string
```

Comment:
This will generate a syntax error. You need a comma, or just leave it on one line.

Reply:
Also, I don't see a need to split this in multiple lines just for aesthetic reasons. We should only make changes that we need. if the parameter list ever gets out of hand, we most likely need to split up a function into multiple smaller ones, but that is not the case here.